src/App.tsx

```diff
@@ -48,20 +48,35 @@ function App() {
     case "in-game": {
       return (
         <div>
-          <div>Goal: {state.goal}</div>
+          <div>Goal: {state.scrabmledGoal}</div>
           <label>
             Guess:
             <input
+              className="uppercase "
               type="text"
-              value={state.guess.trim()}
+              value={state.guess}
               onChange={(ev) =>
                 dispatch({
                   type: "update-guess",
                   newGuess: ev.target.value.toUpperCase(),
+                  skippedWord: "",
                 })
               }
             />
           </label>
+          <div>
+            <button
+              onClick={(e) =>
```

Comment:
since you're not using the event you can omit it

```suggestion
              onClick={() =>
```